ramp-database/rampdb/model/event.py

```diff
@@ -25,6 +25,87 @@
 
 
 class Event(Model):
+    """Event table.
+
+    This table contains all information of a RAMP event.
+
+    Parameters
+    ----------
+    problem_name : str
+        The name of the problem.
+    name : str
+        The name of the event.
+    event_title : str
+        The title to give for the event.
```

Comment:
used in the frontend? this one can contains spaces and the `name` not?

Reply:
Yes you are right